app/views/categories/index.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="flex flex-col h-screen justify-between w-max">
+  <div>
+    <div class="navigation py-6 px-3 blue-nav-white-text">
+      <ul class="flex justify-between items-center w-max1">
+        <li class="w-10">
+          <%= link_to image_tag('backarrow.png', alt: 'Back'), root_path, class: 'nav-link' %>
+        </li>
+        <h2><strong>Categories</strong></h2>
+        <li>
+          <%= link_to 'Sign Out', destroy_user_session_path, data: { turbo_method: :delete }, class: "nav-link text-green" %>
+        </li>
+      </ul>
+    </div>
+
+    <section class="px-5">
+      <p style="color: green"><%= notice %></p>
+
+      <div id="categories">
+        <% @categories.each do |category| %>
+          <a href="<%= category_path(category) %>" class="m-1">
+            <%= render "shared/category", category: category %>
+          </a>
+        <% end %>
+      </div>
+    </section>
+  </div>
+
+  <section class="text-center mt-4">
+    <%= link_to new_category_path do %>
+      <div class="w-full green-btn flex">
+        <button class="p-2 py-4 m-4 text-lg w-full border-2 green-btn rounded hover:bg-green-600 focus:outline-none focus:bg-green-600">
+          <strong>Add a New Category</strong>
+        </button>
+      </div>
+    <% end %>
+  </section>
```

Comment:
- [ ] Your design is good, but there are some minor changes that should be done on the categories and the transactions pages respectively. The add categories button should touch the two ends (left and right), the same should be done on the categories.


![button](https://github.com/Kel-nana/Budget-app/assets/53124548/5e07516e-0089-4099-a5fe-3d628bc2156b)


![spaces](https://github.com/Kel-nana/Budget-app/assets/53124548/4cc22279-8ec1-490a-8f0d-55610d44e0fe)